tools/Environments/DevHome.Environments/DevHome.Environments.csproj

```diff
@@ -28,7 +28,6 @@
     <Folder Include="CustomControls\" />
     <Folder Include="Extensions\" />
     <Folder Include="Strings\" />
-    <Folder Include="ViewModels\" />

```

Comment:
Not needed?

Reply:
This was removed by visual studio but upon looking online looks like how the <Folder Include=.../> is used in this csproj file does nothing, as the folders are included by default. I checked the rest of Dev Home and they don't actually include the folders in the csproj file. Usually what we're worried about in terms of folders in the Csproj files are folders that contain content files like images and other assets as we want them to go to the output directory.